bin/fissile

```diff
@@ -10,12 +10,13 @@ if [ -f ${FISSILE_GIT_ROOT}/Jenkinsfile ] ; then
     # add the common part, which references them.
 
     if [ "${USE_SLE_BASE}" = "false" ]; then
-	cp "${FISSILE_LIGHT_OPEN42}" "${FISSILE_LIGHT_OPINIONS}"
+	SRC_BASE="${FISSILE_LIGHT_OPEN42}"
     else
-	cp "${FISSILE_LIGHT_SLE12}" "${FISSILE_LIGHT_OPINIONS}"
+	SRC_BASE="${FISSILE_LIGHT_SLE12}"
     fi
 
-    cat "${FISSILE_LIGHT_COMMON}" >> "${FISSILE_LIGHT_OPINIONS}"
+    cat "${SRC_BASE}" "${FISSILE_LIGHT_COMMON}" > "${HOME}/$$"
+    mv "${HOME}/$$" "${FISSILE_LIGHT_OPINIONS}"
```

Comment:
Hmm, can we use a `mktmp` instead of dumping random things in `$HOME`? Mostly thinking about the case of (accidentally?) running things on the host instead of the vagrant box.  Let me know if this seems like a dumb idea :D